neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -203,10 +203,11 @@ def feed_dict(self, dataset: Dataset, train: bool = False) -> FeedDict:
         images = np.array(dataset.get_series(self.data_id))
 
         f_dict = {}
-        f_dict[self.image_input] = images / 225.0
-        # it is one everywhere where non-zero, i.e. zero columns are masked out
-        f_dict[self.image_mask] = np.sum(
-            np.sign(images), axis=3, keepdims=True)
+        f_dict[self.image_input] = images / 255.0
+        # the image mask is one everywhere where the image is non-zero, i.e.
+        # zero pixels are masked out
+        f_dict[self.image_mask] = np.sign(
+            np.sum(images, axis=4, keepdims=True))
```

Comment:
moment moment tohle bylo dobře ta axis=3, číslujou se od nuly

Reply:
Ale nevim, proč se maj keepovat dimy

Reply:
Dimy se keepujou, aby to bylo dohro 4D a dal se na tom volat ten pooling a dal se tím násobit obrázek taky.